packages/components/src/theme/ComposerTheme/Components/Slider.ts

```diff
@@ -0,0 +1,17 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Slider: ComponentStyleConfig = {
+  defaultProps: {
+    variant: 'default',
+  },
+  variants: {
+    default: {
+      bg: 'primary-default',
+      sliderBg: 'primary-disabled',
+      boxSize: 3,
+      _focus: {
+        boxShadow: `0px 0px 0px 3px rgba(194, 194, 255, 1)`,
+      },
+    },
+  },
```

Comment:
do we need this if there's only 1 variant?

Reply:
@tkanzakic I can remove the default variant and use the baseStyle key to try set these styles as alternative. Is it acceptable?

Reply:
sounds like a better approach to me, what do you think?

Reply:
@tkanzakic Sounds great! Changes were pushed into the branch.